nvme-print-json.c

```diff
@@ -434,6 +434,16 @@ void json_nvme_id_ctrl(struct nvme_id_ctrl *ctrl,
 	obj_add_int(r, "nmcmr", le16_to_cpu(ctrl->nmcmr));
 	obj_add_int(r, "mcdqpc", le16_to_cpu(ctrl->mcdqpc));
 
+	obj_add_uint(r, "cmmrtd", le16_to_cpu(ctrl->cmmrtd));
+	obj_add_uint(r, "nmmrtd", le16_to_cpu(ctrl->nmmrtd));
+	obj_add_uint(r, "minmrtg", ctrl->minmrtg);
+	obj_add_uint(r, "maxmrtg", ctrl->maxmrtg);
+	obj_add_uint(r, "trattr", ctrl->trattr);
+	obj_add_uint(r, "mcumq", le16_to_cpu(ctrl->mcudmq));
+	obj_add_uint(r, "mcmr", le16_to_cpu(ctrl->mcmr));
+	obj_add_uint(r, "nmcmr", le16_to_cpu(ctrl->nmcmr));
+	obj_add_uint(r, "mcdqpc", le16_to_cpu(ctrl->mcdqpc));
```

Comment:
the nvme-print*.c updates should be a separate patch.

Reply:
Glad you pointed this out because it's already included in #2607. Surprised I didn't get a merge conflict.